recipes/r-qgg/meta.yaml

```
@@ -0,0 +1,123 @@
+{% set version = '1.1.6' %}
+
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-qgg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/qgg_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/qgg/qgg_{{ version }}.tar.gz
+  sha256: dff06a1f4684622a4e60f617758254d25210f5081d814bc143906cea8b297a19
+
+build:
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  
+  ignore_run_exports:
+    - r-data.table
+    - r-corpcor
+    - r-rcpparmadillo
+    - r-matrix
+    - r-rcpp
+    - r-statmod
+    - r-mass
+    - r-mcmcpack
+    - r-coda
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - {{ stdlib("c") }}
+    - {{ stdlib("m2w64_c") }}
+
+  host:
+    - r-base
+    - r-mass
+    - r-mcmcpack
+    - r-matrix
+    - r-rcpp >=1.0.5
+    - r-rcpparmadillo
+    - r-coda
+    - r-corpcor
+    - r-data.table
+    - r-statmod
+    - libopenblas 
+    - libgfortran
```

```suggestion
    - libgfortran
```
I think this is no longer needed either, and is one of the reasons for some of the failures